Sources/PackageManagerDocs/Documentation.docc/IntroducingPackages.md

```diff
@@ -4,8 +4,48 @@ Learn to create and use a Swift package.
 
 ## Overview
 
-overview content here....
+A package consists of Swift source files, including the `Package.swift` manifest file. 
+The manifest file, or package manifest, defines the package's name and its contents using the PackageDescription module.
 
-### First Section
+Each package declares `Products`, a list of what the package produces.
+Types of products include libraries, executables, and plugins:
 
-First section content
+- A library defines a module that can be imported by other Swift code. 
+- An executable is a program that can be run by the operating system.
+- A plugin is executable code that the Swift Package Manager may use to provide additional commands or build capabilities.
+
+The package can also declare `Dependencies`, most frequently other swift packages that provide modules you use.
+Dependencies can also provide macros, plugins, or reference system or binary (non-source) dependencies.
+
+Each product is made up of one or more `Targets`, the basic building block of a Swift package.
+Each target specifies an output, may declare one or more dependencies on other targets within the same package and on products vended by the package’s dependencies.
+A target may define a library, a test suite, an executable, an macro, a binary dependency, and so on.
+
+### About Modules
+
+Swift organizes code into _modules_. 
```

Comment:
Modules also apply to non-Swift code, might be good to work that in somehow.

Reply:
@jakepetroules Do you have - or know of any - good general "what is a module" content? I was really struggling with what the boundaries are. Is module in this context the same as an LLVM module?

Reply:
Multiple, overlapping definitions :) @artemcm might be a good person to help provide more fine grained detail.

Reply:
I tried adding a sentence on the trailing end of that introduction to modules to make it clear that modules could come from any language, but stopped short of asserting any "Module as from LLVM" kind of thing. There's more implication of this in the second paragraph - that the system can provide modules such as Darwin and/or Glibc, which are implied not Swift - but it's fairly indirect. See what you think of this updated content.

The rest of Swift doesn't really talk about modules directly all that much. There's some reference in https://docs.swift.org/swift-book/documentation/the-swift-programming-language/accesscontrol#Modules-Source-Files-and-Packages, but it more jumps to talking about them as they're exposed as "Frameworks"